src/node/abort.cpp

```diff
@@ -16,14 +16,14 @@
 
 namespace node {
 
-void AbortNode(util::SignalInterrupt* shutdown, std::atomic<int>& exit_status, const std::string& debug_message, const bilingual_str& user_message)
+void AbortNode(util::SignalInterrupt* shutdown, std::atomic<int>& exit_status, const bilingual_str& message)
 {
-    SetMiscWarning(Untranslated(debug_message));
-    LogPrintf("*** %s\n", debug_message);
-    InitError(user_message.empty() ? _("A fatal internal error occurred, see debug.log for details") : user_message);
+    SetMiscWarning(message);
+    LogError("%s\n", message.original);
+    InitError(_("A fatal internal error occurred, see debug.log for details: ") + message);
```

Comment:
While this code is touched, the `LogError` call seems redundant as the following `InitError` logs the `message` unconditionally:
```
2024-03-21T12:08:54Z [error] Flushing block file to disk failed. This is likely the result of an I/O error.
2024-03-21T12:08:54Z Error: A fatal internal error occurred, see debug.log for details: Flushing block file to disk failed. This is likely the result of an I/O error.
```

Reply:
> It's the uiInterface that logs it. I think if you run `bitcoin-qt`, there should be no redundant log line.

When running the GUI, the log line redundancy remains :)